tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -45,10 +52,14 @@ export class GraphExecutor implements FunctionExecutor {
   private _functions: {[key: string]: Graph} = {};
   private _functionExecutorMap: {[key: string]: FunctionExecutor} = {};
   private _resourceManager: ResourceManager;
-  private intermediateTensors: NamedTensorsMap = {};
-  private keepIds: Set<number>;
+  // Variables with Async suffix is used for dumping by executeAsync.
+  private tensorsPendingDisposalForAsync: NamedTensorsMap;
+  private idsKeepForAsync: Set<number>;
+  // Variables with Sync suffix is used for dumping by execute.
+  private tensorsPendingDisposalForSync: Tensor[];
```

Comment:
Can you unify to use `tensorsPendingDisposal` or `intermediateTensorsPendingDisposal` for both async and sync for simplicity?

Reply:
Done